lib/widgets/sample_viewer_page.dart

```diff
@@ -42,6 +44,35 @@ class _SampleViewerPageState extends State<SampleViewerPage> {
   var _ready = false;
   var _searchHasFocus = false;
 
+  final _searchPrefixes = [
+    'Try',
+    'Look for',
+    'Search',
+    'Explore',
+    'Type to explore',
+    'Check out',
+    'Discover',
+    'Start typing',
+    'Tap to search',
```

Comment:
```suggestion
```
I think the "tap to search" instruction is a little confusing as it feels like the user should tap on the suggestion to search for it. The "Start typing" prefix in the list is a better expectation. The others look good too.